src/stories/Icons.stories.tsx

```diff
@@ -128,14 +138,17 @@ const icons = pick(allImports, [
   'StarIcon',
   'StatusIpIcon',
   'StatusOkIcon',
+  'StopIcon',
   'SuccessIcon',
   'SuitcaseIcon',
   'TableIcon',
   'TagIcon',
   'TerminalIcon',
+  'ThumbsDownIcon',
```

Comment:
Useless because we. can do `transform="scale(1, -1)" on `ThumbsUpIcon`

Reply:
You decide wether to remove it or not ^^ won't force you with a red review.

Reply:
I think it's worth the convenience to have it in there, but I did replace the component with a transformed version of `ThumbsUpIcon` to at least ensure consistency between the two and save on bundle size. Same with `FastReverseIcon`.